WcaOnRails/config/application.rb

```diff
@@ -14,6 +14,17 @@
 ENVied.require(*ENV['ENVIED_GROUPS'] || Rails.groups)
 
 module WcaOnRails
+  AVAILABLE_LOCALES = {
```

Comment:
I put this variable here because I need it in this configuration file and I think rails models are not loaded yet at this point.
So 2 questions:
 - Is it the right place?
 - Should we load this from some json file in the config directory instead of hardcoding the values?